src/sizes.rs

```diff
@@ -818,6 +818,8 @@ mod extra_sizes {
     pub type U35664 = uint!(0 0 0 0 1 0 1 0 1 1 0 1 0 0 0 1);
     pub type U49856 = uint!(0 0 0 0 0 0 1 1 0 1 0 0 0 0 1 1);
 
+    //NTRU-PRIME sizes
+    pub type U1277 = uint!(1 0 0 1 1 1 1 1 1 0 1);
```

Comment:
Can you keep this in numerical order? (i.e. place it above "ML-DSA sizes") Thanks